src/main/java/com/minecolonies/coremod/items/ItemBuildTool.java

```diff
@@ -42,7 +42,7 @@ public EnumActionResult onItemUse(
                                        final float hitY,
                                        final float hitZ)
     {
-        playerIn.addStat(ModAchievements.achievementWandOfbuilding);
+        //playerIn.addStat(ModAchievements.achievementWandOfbuilding);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)